datalake/src/main/java/com/sequenceiq/datalake/flow/create/event/EnvWaitRequest.java

```diff
@@ -2,19 +2,27 @@
 
 import com.sequenceiq.datalake.flow.SdxContext;
 import com.sequenceiq.datalake.flow.SdxEvent;
+import com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.responses.DatabaseServerStatusV4Response;
 
 public class EnvWaitRequest extends SdxEvent {
 
-    public EnvWaitRequest(Long sdxId, String userId) {
+    private DatabaseServerStatusV4Response databaseServerResponse;
```

Comment:
passing this response along is not necessary. If I'm not mistaken it's used in `ProvisionerService#startStackProvisioning`, but that info is already in `SdxCluster`, so I would drop it entirely if possible